### ISSUE TITLE
Question: Are there plans to add a Firefox version of the extension?

### ISSUE THREAD

**Issue by jish** [jayehernandez/letra-extension#410]
**Is your feature request related to a problem? Please describe.**

No not necessarily a problem, just that I can't install the Letra extension in Firefox.

**Describe the solution you'd like**

It would be nice if the build process built both a Chrome version of the extension as well as a Firefox version.

**Describe alternatives you've considered**

I have considered attempting to one-off research, build, and install the Firefox extension on my own, but I believe this would not help any other Firefox users. I also wanted to ask first to see if anyone else has investigated building a Firefox version so I do not duplicate any work that has already been done.

**Additional context**

Here is a page on building a cross-browser extension: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Build_a_cross_browser_extension

**Comment by Kvaibhav01** [comment]
Thanks for opening the issue @jish! I agree that we should have a cross-browser extension but as far as I know, we're not into Firefox extension as of now.

@jayehernandez currently there is no plan for Firefox extension development right?

**Comment by jayehernandez** [comment]
Yup, currently there’s no plan for developing a Firefox extension. But I think this was also discussed before with @Kvaibhav01 and it is something I’m open too! (Just don’t have the time to research now due to lack of time. 😅) This would open up the extension to more users.

@jish no one has investigated on working on the Firefox extension yet — would you like to start?

**Comment by AlexxMart** [comment]
I would like to give a hand and make this work on Firefox

**Comment by rookiemonkey** [comment]
Good day @jayehernandez i submitted a pull request for this issue. I hope I was able to help ☺️

**Comment by gugacavalieri** [comment]
Hi guys! Has this been approved by the Firefox team yet? :smile:

**Comment by jayehernandez** [comment]
Hey @gugacavalieri! (and everyone else on here) They disabled the extension as there were differences in the `main.js` file when they tried to compile it on their end. When I checked their `main.js` file with mine, the changes were just the **different variables** used when compiling Vue.js code. They asked me if I could try hardcoding these variables but that was done by the compiler so I'm not really sure if there's a way to that?

I did this before the holiday season started last year, and it was automatically rejected after I asked if there was any other way to prove that the code is the same with their compiled version.

For reference, here's a sample of their `main.js` file
```
function Be(t,e){return(t.__esModule||it&&"Module"===t[Symbol.toStringTag])&&(t=t.default),s(t)?e.extend(t):t}function ze(t){return t.isComment&&t.asyncFactory}function Ue(t){if(Array.isArray(t))for(var e=0;e<t.length;e++){var a=t[e];if(o(a)&&(o(a.componentOptions)||ze(a)))return a}}function Ve(t,e){qe.$on(t,e)}function He(t,e){qe.$off(t,e)}function Ge(t,e){var a=qe;return function n(){var o=e.apply(null,arguments);null!==o&&a.$off(t,n)}}f
```

And here's my `main.js`
```
function Be(t,e){return(t.__esModule||it&&"Module"===t[Symbol.toStringTag])&&(t=t.default),s(t)?e.extend(t):t}function Ue(t){return t.isComment&&t.asyncFactory}function Ve(t){if(Array.isArray(t))for(var e=0;e<t.length;e++){var a=t[e];if(o(a)&&(o(a.componentOptions)||Ue(a)))return a}}function ze(t,e){qe.$on(t,e)}function He(t,e){qe.$off(t,e)}function Ge(t,e){var a=qe;return function n(){var o=e.apply(null,arguments);null!==o&&a.$off(t,n)}}
```

So for the code above, their version had `function Ue(t)` while mine had `function Ve(t)`. The whole file (400k characters) has the same problems as this piece of code. Does anyone know if this is a compiler problem? I haven't had this issue when submitting to Chrome's store.
